src/test_servers.py

```diff
@@ -12,18 +12,30 @@
 ]
 
 
-@pytest.mark.parametrize("result", PARAMS_TABLE)
-def test_message_completion(result):
-    """Test table of potential messages for receipt and transmission."""
-    from client import client
-    assert client(result) == result
+# @pytest.mark.parametrize("result", PARAMS_TABLE)
+# def test_message_completion(result):
+#     """Test table of potential messages for receipt and transmission."""
+#     from client import client
+#     assert client(result) == result
 
 
-def test_message_unicode():
-    """Test for unicode messages."""
-    from client import client
-    msg = 'CÅT'
-    if sys.version_info[0] == 2:
-        assert client(msg) == msg.decode('utf8')
-    else:
-        assert client(msg) == msg
+# def test_message_unicode():
+#     """Test for unicode messages."""
+#     from client import client
+#     msg = 'CÅT'
+#     if sys.version_info[0] == 2:
+#         assert client(msg) == msg.decode('utf8')
+#     else:
+#         assert client(msg) == msg
+
+
+def test_response_ok():
```

Comment:
These tests are just testing the content of your response functions. You still need to assert that your server sends back the expected response.